ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -34,6 +38,17 @@ function leftPad(numToPad, size) {
     .slice(-size)
 }
 
+function massageData(attributeName, attributeValue) {
+  if (metacardDefinitions.metacardTypes[attributeName].type === 'DATE') {
+    return (
+      attributeName.toUpperCase() +
+      ': ' +
+      Common.getHumanReadableDateTime(attributeValue)
+    )
+  }
+  return attributeName.toUpperCase() + ': ' + attributeValue
+}
+
```

Comment:
✏️ Perhaps this would read better as something like:
```function massageData(attributeName, attributeValue) {
  if (metacardDefinitions.metacardTypes[attributeName].type === 'DATE') {
       attributeValue = Common.getHumanReadableDateTime(attributeValue)
  }
  return attributeName.toUpperCase() + ': ' + attributeValue
}```

That way you don't have the repeated code, and you get rid of the multi-line return. 
A ternary operator would work here in theory, but I think it would be messy.